.github/workflows/make_wheel.yml

```diff
@@ -43,6 +43,7 @@ jobs:
         cfg:
           - { name: 'Linux LLVM+libstdc++', os: 'ubuntu-20.04', platform: 'manylinux_2_31_x86_64', cc: clang, cxx: clang++, config: }
           - { name: 'MacOS LLVM+libc++', os: 'macos-10.15', platform: 'macosx_10_15_x86_64', cc: clang, cxx: clang++, config: --config=libc++ }
+          - { name: 'MacOS LLVM+libc++', os: 'macos-11', platform: 'macosx_10_16_x86_64', cc: clang, cxx: clang++, config: --config=libc++ }
         py:
           - { version: '3.6', interp: 'cp36', abi: 'cp36m' }
```

Comment:
Does Py 3.6 still exist on macos-11?

Reply:
I think macos 11 defaults to Python 3.8.

Reply:
Well, the question is whether this buil tinstruction as written will actually succeed on the GitHub action runner. The macos-11 documentation suggests that only versions 3.7 to 3.10 are available.